### PR TITLE
CIAM-2404 Ensure using the latest versions of RPM packages by calling microdnf update

### PR DESCRIPTION
**Pull request by drichtarik**
@iankko

### CODE REVIEW

File: modules/sso/sso-pre-launch-actions/configure.sh [hunk 1]
```diff
@@ -5,5 +5,8 @@ set -eu
 # shellcheck disable=SC1091
 source "${JBOSS_HOME}/bin/launch/sso-rcfile-definitions.sh"
 
+# CIAM-2404 Ensure using the latest versions of microdnf packages
```
Comment (iankko) [inline modules/sso/sso-pre-launch-actions/configure.sh]: Please change the comment to read `Ensure using the latest versions of RPM packages by calling microdnf update`
(the change is not about using the latest version of microdnf packages, but actually about performing RPM packages upgrades to latest available versions)

**Review by iankko — CHANGES_REQUESTED** [review]
@drichtarik  LGTM. But small change request - would it be
possible to change the commit message and also the comment
to say about RPM package upgrades, instead of microdnf package updates?

**Comment by drichtarik** [comment]
@iankko done

**Review by iankko — APPROVED** [review]
LGTM AFAICT, thanks Denis!

**Pull request merged**